app/javascript/dashboard/components/widgets/conversation/ConversationBasicFilter.vue

```diff
@@ -85,6 +89,15 @@ export default {
     },
     onChangeFilter(type, value) {
       this.$emit('changeFilter', type, value);
+      this.saveSelectedFilter(type, value);
+    },
+    saveSelectedFilter(type, value) {
```

Comment:
The `type` and `value` seems a bit odd here, seems like `type` is holding the actual value. Let's fix it upstream if required